### ISSUE TITLE
"Unable to locate package python-certbot-nginx"

### ISSUE THREAD

**Issue by BigSnicker** [Universidad-de-Costa-Rica/role-mailman3#1]
Beautiful work. Have been struggling with getting a mailman3 server up and just discovered ansible, so I'm hoping I have better luck here with your work.

But my first attempt to run it on Debian 10 resulted in a failure at the certbot installation, saying it was unable to locate python-certbot-nginx, which appears to be deprecated and not obvious to install.  Attempted to use python3-certbot-nginx instead, but didn't have much luck.

**Comment by BigSnicker** [comment]
I should mention that I was doing the installation on a fresh install of Deb 10 on a dedicated server, so it's a very simple environment and configuration.

**Comment by jobcespedes** [comment]
Hi,

Package `python-certbot-nginx` should be available in buster according to [this](https://packages.debian.org/buster/python-certbot-nginx). Maybe, an `apt update` is needed in an fresh install prior pkg install.

**Comment by jobcespedes** [comment]
@valarauco what do yout think?

**Comment by BigSnicker** [comment]
I'm going to retest this using the digitalocean server and will post the results here.
The workaround, by the way, was just to do the certbot run after ansible.

**Comment by valarauco** [comment]
Hi BigSnicker!

Thank you for helping us testing this project! 

@jobcespedes I'll try to reproduce the issue, nevertheless it seams this package is being replaced soon and we have to upgrade it.

**Comment by valarauco** [comment]
I just pushed some fixes I found.

fcf6d969591008d91a65714a861c7e2d3b28423a: Adds cache_valid_time to apt tasks in case the cache was not valid and packages were not found.

f4e3b8a1fd84f646ef38e46e1b639bef3afd3212: Nginx's config backups are created in  sites-enebled folder, so they were crashing my configs, so I made a workaround. Maybe we can find a better solution.

a24f46b817e9301ca7bf218f9ec672b5d9e24a7f: I think variable service_name was wrong and changed to mailman3_service_name which is in the defaults.

I tested this changes in a new virtual server with Debian 10.7 and public IP with successful results. Not tested on Digital Ocean or similar.
